src/modules/bill-runs-import/lib/queries.js

```diff
@@ -76,5 +77,193 @@ join water.billing_invoices i on concat_ws(':', nbh."FGAC_REGION_CODE", nbh."ID"
 left join import."NALD_BILL_TRANS" nbt on nbh."FGAC_REGION_CODE"=nbt."FGAC_REGION_CODE" and nbh."ID"=nbt."ABHD_ID" 
 join import."NALD_ABS_LICENCES" nl on nbt."FGAC_REGION_CODE"=nl."FGAC_REGION_CODE" and nbt."LIC_ID"=nl."ID"
 left join water.licences l on nl."LIC_NO"=l.licence_ref
-on conflict (billing_invoice_id, licence_id) do nothing
+on conflict (billing_invoice_id, licence_id) do nothing;
```

Comment:
I'm assuming this will only ever run once on production so not attempting any upsert functionality (on other environments we will need to wipe the tables and re-import)